src/cli/task-info.ts

```diff
@@ -13,9 +13,9 @@ export const taskInfo = (coreCompiler: CoreCompiler, sys: CompilerSystem, logger
 
   console.log(``);
   console.log(`${logger.cyan('      System:')} ${sys.name} ${sys.version}`);
-  console.log(`${logger.cyan('    Platform:')} ${details.platform} (${details.release})`);
+  console.log(`${logger.cyan('    Platform:')} ${details?.platform} (${details?.release})`);
```

Comment:
to avoid having `undefined` in the logs
```suggestion
  if (details) {
    console.log(`${logger.cyan('    Platform:')} ${details?.platform} (${details?.release})`);
  }
```

Reply:
I thought about doing something like this, however I don't think it will ever be undefined in practice. In the `cli/` module we're always operating with a node-specific `CompilerSystem`, on which the `details` property is always defined:

https://github.com/ionic-team/stencil/blob/654d7213c1c21647b5df83c14b90fdeed3ed47f1/src/sys/node/node-sys.ts#L652-L661

so I don't think in practice it will ever happen. I'd like to change the type to more accurately reflect the fact that it's always defined, but this is a public type so that would be a breaking change unfortunately

Reply:
anyway - added a check in just for the heck of it